src/base/bittorrent/session.h

```diff
@@ -67,12 +64,15 @@ namespace Net
 
 namespace BitTorrent
 {
+    class CacheStatus;
     class InfoHash;
     class MagnetUri;
     class ResumeDataStorage;
     class SessionImpl;
```

Comment:
This one was used in initial PIMPL variant. I missed to remove it. Please do it.

Reply:
Done.